### ISSUE TITLE
Cannot change my username

### ISSUE THREAD

**Issue by theleoad** [jamesctucker/teach-yourself-code#16]

**Comment by ksorv** [comment]
@jamesctucker Is that a feature we're aiming for?

**Comment by theleoad** [comment]
I use G Suite, and my main email is talkto@theleoad.com. The app uses the "first part" of the email to generate the username. So my username is talkto. 
I think it would be nice to change the username. =]

**Comment by jamesctucker** [comment]
@theleoad @ksorv Yes, that's something we should definitely have available for users. I wonder if that's something native to Auth0 or if we'll have to create a custom solution.

**Comment by ksorv** [comment]
Yay! They do support changing usernames!

Doc link attached [here.](https://auth0.com/docs/api/management/v2?_ga=2.64662317.362506009.1588723242-236529937.1588723242#!/Users/patch_users_by_id)
